contracts/interchain-token-service/src/contract.rs

```diff
@@ -50,6 +50,19 @@ impl InterchainTokenService {
             &interchain_token_wasm_hash,
         );
     }
+
+    fn set_token_id_config(env: &Env, token_id: BytesN<32>, token_data: TokenIdConfig) {
+        env.storage()
```

Comment:
Let me know if we want to make this fn public and add `Self::owner(env).require_auth();`

Reply:
The owner shouldn't have that kind of privilege to arbitrarily set/overwrite a token id config. The config is created only on deployments for correct ITS behaviour